src/padplus-manager/api-request/dedupeApi.ts

```diff
@@ -0,0 +1,143 @@
+import { DelayQueueExecutor } from 'rx-queue'
+import { log } from '../../config'
+import { ApiType } from '../../server-manager/proto-ts/PadPlusServer_pb'
+
+// Expire time for api call data that persist in the pool
+// Number of seconds
+const EXPIRE_TIME = 10
+
+const DEDUPE_API = [
+  ApiType.ADD_CONTACT,
```

Comment:
Why do we need to dedupe this api?

Reply:
It could only request once per 10 seconds, maybe useful for limit user operation.

Reply:
The logic should be that, the same request (same api with same parameters) can only be emit once in every 10 seconds. If this is the case, you . can not limit user operation if the user try to add multiple person at the same time.

Reply:
OK, I will remove this type.